app/lib/shared/redis_cache.dart

```diff
@@ -247,6 +247,16 @@ class CachePatterns {
         decode: (d) => CountData.fromJson(d as Map<String, dynamic>),
       ))[package];
 
+  Entry<WeeklyDownloadCounts> weeklyDownloadCounts(String package) => _cache
+      .withPrefix('weekly-download-counts/')
+      .withTTL(Duration(hours: 6))
+      .withCodec(utf8)
+      .withCodec(json)
+      .withCodec(wrapAsCodec(
+        encode: (WeeklyDownloadCounts wdc) => wdc.toJson(),
+        decode: (d) => WeeklyDownloadCounts.fromJson(d as Map<String, dynamic>),
```

Comment:
By default it's possible to store binary data in redis.

That's why we have `.withCodec(utf8)`.

So why not just store:
```dart
base64.encode(Uint8List.sublistView(Uint32List.fromList(dataPoints)));
```
Or:
```dart
return Uint8List.sublistView(Uint32List.fromList(dataPoints));
```